services/drupal/web/themes/epa_theme/source/_patterns/05-components/box/box--aligned-right-stacked/box--aligned-right-stacked.md

```diff
@@ -0,0 +1,25 @@
+---
+el: .box.is-aligned-right.clear-right
```

Comment:
Although we added `clear-right` to the CSS, we prefer to use the `u-clear-right` class to make it obvious we’re using a utility class.

Also, `is-aligned-right` should be `u-align-right`.